bin/setup_mongo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "Start MongoDB without access control."
+mongod --port 27017 &
+while ! nc -z 127.0.0.1 27017; do sleep 1; done
+
+
+echo "Create the user administrator."
+mongo admin --eval "db.createUser({ user: '${MONGO_USER}', pwd: '${MONGO_PASSWORD}', roles: [ { role: 'dbAdminAnyDatabase', db: 'admin' }, { role: 'clusterAdmin', db: 'admin' } ] });"
+
+
+echo "Shutdown the MongoDB service."
+mongod --shutdown
+
+
+echo "Creating keyFile for replication."
+echo -e ${MONGO_KEY} > /etc/mongod.key
+chmod 400 /etc/mongod.key
+
+echo "Overwrite setup_mongo.sh so that this is a one-time setup"
+echo "#!/bin/bash" > ./setup_user.sh
```

Comment:
Assuming this overwrote itself (`setup_mongo.sh`), this would only work on restarts of the current container, but would not prevent a second user creation when a user reuses an old data directory (like `docker-compose` when run on non-triton).  In looking to add similar functionality to the mongo official image (https://github.com/docker-library/mongo/pull/53#issuecomment-275257908), we didn't yet have a way to detect that this was an already initialized database directory to prevent trying to create the user a second time.